public/app/features/trails/MetricScene.tsx

```diff
@@ -40,21 +41,23 @@ export interface MetricSceneState extends SceneObjectState {
   body: MetricGraphScene;
   metric: string;
   actionView?: string;
+  layout: LayoutType;
```

Comment:
Any specific reason why you added this to the MetricScene and not just added URL sync to LayoutSwitcher?